Index: src/Lesson4.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>public class Lesson4 {\r\n    public static void main(String[] args) {\r\n        System.out.println(\"Testing\");\r\n    }\r\n}\r\n
===================================================================
diff --git a/src/Lesson4.java b/src/Lesson4.java
--- a/src/Lesson4.java	
+++ b/src/Lesson4.java	
@@ -1,5 +1,5 @@
-public class Lesson4 {
-    public static void main(String[] args) {
-        System.out.println("Testing");
-    }
-}
+ public class Lesson4 {
+     public static void main(String[] args) {
+
+     }
+}
\ No newline at end of file
